addons/website/static/src/systray_items/edit_website.js

```diff
@@ -23,6 +23,9 @@ class EditWebsiteSystray extends Component {
         useEffect((snippetsLoaded) => {
             if (snippetsLoaded) {
                 this.state.isLoading = false;
+                // Once the sinppet is loaded, set the focus on the search input.
+                this.snippetSearchInput = document.querySelector('.o_snippet_search_filter_input');
```

Comment:
@bvr-odoo Its better i should do  `document.querySelector('.o_snippet_search_filter_input'),focus()`, instead of adding two line, i will change that. Thanks !